src/NLPL.xml

```diff
@@ -0,0 +1,35 @@
+<spdx name="No Limit Public License" identifier="NLPL" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/NLPL</url>
+  </urls>
+  <license>
+    <title>
+      <p>NO LIMIT PUBLIC LICENSE 
+        <br/>Version 0, June 2012 
+      </p>
+    </title>
+    <copyright>
```

Comment:
I don't think this should be labeled with copyright tag.  Not sure it should be tagged at all??